task.sql

```diff
@@ -1 +1,5 @@
--- write your code for database user creation here
\ No newline at end of file
+CREATE USER 'webappuser'@'%' IDENTIFIED BY 'P@ssw0rd';
+GRANT INSERT, SELECT, UPDATE, DELETE ON ShopDB.* TO 'webappuser'@'%';
+
+CREATE USER 'deploymentuser'@'%' IDENTIFIED BY 'P@ssw0rd';
+GRANT ALL ON ShopDB.* TO 'deploymentuser'@'%';
```

Comment:
![image](https://github.com/user-attachments/assets/62b30187-bbfa-41e0-89a5-b0b4ae55bff3)
Add Eof(End-of-file), need to add a blank line at the end